internal_ws/yktrace/src/swt.rs

```diff
@@ -110,6 +120,8 @@ unsafe extern "C" fn __yk_swt_rec_loc(symbol_name: *const i8, bb_idx: u32) {
     __yk_swt_rec_loc_impl(symbol_name, bb_idx);
 }
 
+// FIXME this doesn't export __yk_swt_rec_loc from the dylib
+/*
```

Comment:
This is all commented out?

Reply:
Yes, it is currently broken for the reason stated by the FIXME. Once it is fixed again, this and the `#[cfg]` above you commented on should be uncommented again.